.tool-versions

```diff
@@ -1 +1 @@
-elixir 1.13.1
```

Comment:
This can be changed back. I just didn't want to install that version while working on this PR. Although I think it makes sense to use the latest possible version during development :)

Reply:
I have not yet tested the code and what will or not break as OTP 26 introduce a change around map atom key ordering to not be garanteed .

not sure the side effect on that yet but maybe a separate MR might be better unless we are confident it s not breaking ?

Reply:
I don't think maps are used in Hammer.

Reply:
yea was just putting it out there as I recently upgraded bunch of app at my work